account_fiscal_month/views/date_range_type.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="date_range_type_form_view">
```

Comment:
Why not keeping same xml id as parent view_date_range_type_form_view and putting the id before the model?